controllers/numaresourcesoperator_controller_test.go

```diff
@@ -787,6 +791,41 @@ var _ = Describe("Test NUMAResourcesOperator Reconcile", func() {
 			Expect(args).ToNot(ContainElement(ContainSubstring("--notify-file")), "malformed args: %v", args)
 		})
 
+		It("should find default behavior of RTE pause disabled", func() {
+			conf := nropv1.DefaultNodeGroupConfig()
+
+			nro := testobjs.NewNUMAResourcesOperatorWithNodeGroupConfig(objectnames.DefaultNUMAResourcesOperatorCrName, &labSel, &conf)
+
+			reconciler := reconcileObjects(nro, mcp)
+
+			key := client.ObjectKeyFromObject(nro)
+
+			nroCurrent := &nropv1.NUMAResourcesOperator{}
+			Expect(reconciler.Client.Get(context.TODO(), key, nroCurrent)).NotTo(HaveOccurred())
+			Expect(nroCurrent.Spec.NodeGroups[0].Config.RTEPause).To(Equal(conf.RTEPause))
+
+			//we do not check the ds here because we don't want to update ds and cause a restart of the pods,
+			// we only pause the pods sync with NRTs
```

Comment:
we should check that the DS spec command line does NOT include the `--no-publish` flag

Reply:
we should have tests which check that if when the setting is given with pause disabled no changes are done